Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -10,18 +10,22 @@ public protocol DerivePartialIsos {}
 public enum Route: DerivePartialIsos {
   case about
   case account
+  case cancel
+  case confirmCancel
```

Comment:
we could start nesting some of these routes inside an `Account` sub route...

Reply:
Oh definitely. I started going down that path but it was too late for me to understand how that works for the first time. Can refactor today!